src/contracts/pods/EigenPod.sol

```diff
@@ -68,16 +76,16 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
 
     // STORAGE VARIABLES
     /// @notice the amount of execution layer ETH in this contract that is staked in EigenLayer (i.e. withdrawn from the Beacon Chain but not from EigenLayer), 
-    uint64 public restakedExecutionLayerGwei;
+    uint64 public withdrawableRestakedExecutionLayerGwei;
 
     /// @notice an indicator of whether or not the podOwner has ever "fully restaked" by successfully calling `verifyCorrectWithdrawalCredentials`.
     bool public hasRestaked;
 
-    /// @notice this is a mapping of validator indices to a Validator struct containing pertinent info about the validator
-    mapping(uint40 => VALIDATOR_STATUS) public validatorStatus;
+    /// @notice This is a mapping of validatorPubkeyHash to withdrawalIndex to whether or not they have proven a withdrawal for that index
+    mapping(bytes32 => mapping(uint64 => bool)) public provenPartialWithdrawal;
```

Comment:
why the switch in order? it's not a problem, just curious

Reply:
like the order of the mappings in storage?  idk how that happened lol.  I can change it back

Reply:
order of these variables in comparison to the contract before

Reply:
by `withdrawal index` you mean `slot number`, right?